examples/factor_analysis/get_tfa_input_from_nifti.py

```diff
@@ -0,0 +1,74 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import scipy.io
+from scipy.stats import stats
+import numpy as np
+import nibabel as nib
+from nilearn.input_data import NiftiMasker
+import os
+import logging
+import sys
+import click
+
+@click.command()
+@click.argument('nifti_file', type=click.File('rb'))
+@click.argument('out_file', type=click.File('wb'))
+@click.option('--mask_file', default=None, type=click.File('rb'), help='The mask file to get ROI')
+@click.option('--zscore', is_flag=True, help='to zscore fMRI data')
+@click.option('--detrend', is_flag=True, help='to detrend fMRI data')
+@click.option('--smoothing_fwmw', default=False, help='the spatial smooth window size')
```

Comment:
The GNU convention (quite popular, see `curl`) is for long command line arguments to use dashes, not underscores:
https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html
Click automatically converts them to underscores:
http://click.pocoo.org/5/parameters/#parameter-names
You should also use dashes.